Movie/Movie/AssemblyBilder.swift

```diff
@@ -0,0 +1,29 @@
+// AssemblyBilder.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Протокол билдера
+protocol AssemblyBilderProtocol {
+    func createMainModule(router: RouterProtocol) -> UIViewController?
+    func createDetailModule(id: String, router: RouterProtocol) -> UIViewController?
+}
+
+/// Билдер
+final class AssemblyBilder: AssemblyBilderProtocol {
+    func createMainModule(router: RouterProtocol) -> UIViewController? {
+        let view = MovieViewController()
+        let networkService = NetworkManager()
```

Comment:
```suggestion
        let networkManager = NetworkManager()
```
или везде сервисы или менеджеры